content/docs/2.3/scalers/azure-event-hub.md

```diff
@@ -30,6 +30,11 @@ triggers:
 - `consumerGroup` - Consumer group of event hub consumer. (default: `$default`)
 - `unprocessedEventThreshold` - : Average target value to trigger scaling actions. (default: 64)
 - `blobContainer` - Container name to store checkpoint. This is needed when a using an Event Hub application written in dotnet or java, and not an Azure function
+- `checkpointStrategy` - configure the checkpoint behaviour of different eventhub sdk's. (default: `""`)
+    - `""`: the default behaviour works for C#, older Python and Java eventhub implementations if a `blobContainer` is configured.
+    - `AzureWebJob`: suitable for Azure Functions. This is the default setting, if `blobcontainer` is empty.
```

Comment:
Now that I read this again, what if we just call this `AzureFunctions`?

Reply:
I've thought about it too. I think the corresponding lib is AzureWebJob. But in this context it is probably only used by Azure Functions. I am going to rename this